src/Common/src/System/Xml/XmlConvertEx.cs

```diff
@@ -66,7 +66,7 @@ public static string[] SplitString(string value)
 
         public static Uri ToUri(string s)
         {
-            if (s != null && s.Length > 0)
```

Comment:
@alexperovich The following two expressions have identical behavior:

* `s?.Length > 0`
* `(s?.Length ?? 0) > 0`

The first comparison is the "lifted to null" version of the operation on two integers, and considers a `null` value unordered with respect to any value (i.e. `default(int?) > 0` is false and so is `default(int?) <= 0`).